ansible/ansible-ai-connect-service
# Containerfile, Makefile and compose.yaml update

Jira Issue: <https://issues.redhat.com/browse/AAP-33560>


## Description


Update Containerfile, Makefile and compose.yaml file to include Chatbot UI React application.

*Note:* JavaScript/TypeScript linting is not included in this PR. It will be covered in the following PRs.

## Testing

### Steps to test
1. Pull down the PR
2. Run
```
make chatbot-bundle
make chatbot-test
make build-wisdom-container
```
to verify each command completes successfully.

### Scenarios tested

See above.

## Production deployment
